src/NumberTheory/GaloisGrp/GaloisGrp.jl

```diff
@@ -894,7 +894,7 @@ function set_orbit(G::PermGroup, H::PermGroup)
   #    http://dblp.uni-trier.de/db/journals/jsc/jsc79.html#Elsenhans17
   # https://doi.org/10.1016/j.jsc.2016.02.005
 
-  l = low_index_subgroup_reps(H, 2*degree(G)^2)
+  l = map(representative, low_index_subgroup_classes(H, 2*degree(G)^2))
```

Comment:
Could also use broadcasting (I am fine with either variant)
```suggestion
  l = representative.(low_index_subgroup_classes(H, 2*degree(G)^2))
```